.github/release-please/config.json

```
@@ -29,6 +29,12 @@
     },
     "images/spilo": {
       "package-name": "spilo"
+    },
+    "images/masterportal/masterportal_V2": {
```

```suggestion
    "images/masterportal/masterportal-v2": {
```
docker images seem to use hyphens and are all lower case. Not sure if that matters anywhere

---

done